src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -0,0 +1,218 @@
+      showPagination: false
+    };
+  }
+
+  setInstitutions = ({ institutionCount, institutionQuery, institutions, pagesCount }) => {
+    if (institutionQuery === this.state.searchInputValue) {
+      this.setState({
+        loadingInstitutions: false,
+        loadingPage: false,
+        showInstitutions: true,
+        institutionCount,
+        institutions,
+        pagesCount,
+        showPagination: pagesCount > 1
+      });
+    }
+  }
+
+  scrollToTop = () => {
+    scroller.scrollTo('schoolSearch', {
+      duration: 250,
+      delay: 0,
+      smooth: true
+    });
+  };
+
+  handleSearchInputKeyDown = e => {
+    if ((e.which || e.keyCode) === 13) {
+      e.preventDefault();
+      e.target.blur();
+
+      this.setState({
+        loadingInstitutions: true,
+        loadingPage: false,
+        page: 1,
+        pageCount: 0,
+        showInstitutions: false,
+        showPagination: false
+      });
+
+      this.debouncedSearchInstitutions({ institutionQuery: this.state.searchInputValue });
+    }
+  }
+
+  handleSearchClick = e => {
+    e.preventDefault();
+    this.setState({
+      loadingInstitutions: true,
+      loadingPage: false,
+      page: 1,
+      pageCount: 0,
+      showInstitutions: false,
+      showPagination: false
+    });
+
+    this.debouncedSearchInstitutions({ institutionQuery: this.state.searchInputValue });
+  }
+
+  handleSearchInputChange = e => {
+    let searchInputValue;
+    if (typeof (e) === 'string') {
+      searchInputValue = e;
+    } else {
+      searchInputValue = e.target.value;
+    }
+    this.setState({
+      searchInputValue
+    });
+  }
+
+  handleOptionClick = facilityCode => {
+    this.props.onChange(facilityCode);
+
+    this.setState({
+      facilityCodeSelected: facilityCode
+    });
+  }
+
+  handlePageSelect = page => {
+    this.scrollToTop();
+
+    this.setState({
+      loadingInstitutions: false,
+      loadingPage: true,
+      page,
+      pageCount: 0,
+      showInstitutions: false,
+      showPagination: false
+    });
+
+    this.debouncedSearchInstitutions({
+      institutionQuery: this.state.searchInputValue,
+      page
+    });
+  }
+  render() {
+    const {
+      formContext
+    } = this.props;
+    /*
+    const fieldsetClass = classNames('fieldset-input', {
+      'usa-input-error': this.props.errorMessage,
+      [this.props.additionalFieldsetClass]: this.props.additionalFieldsetClass
+    });
+
+    const legendClass = classNames('legend-label', {
+      'usa-input-error-label': this.props.errorMessage,
+      [this.props.additionalLegendClass]: this.props.additionalLegendClass
+    });
+    */
+
+    const fieldsetClass = classNames('search-select-school-fieldset');
+    const legendClass = classNames('legend-label');
+
+    if (formContext.reviewMode) {
+      return (<div>To do</div>);
+    }
+
+    return (
+      <fieldset className={fieldsetClass}>
+        <legend className={legendClass}>
+          {'Search school names'}
```

Comment:
![screen shot 2018-07-25 at 17 11 53](https://user-images.githubusercontent.com/4998130/43232412-ddbe71c2-902d-11e8-9cfa-a603f9375f55.png)
![screen shot 2018-07-25 at 17 11 41](https://user-images.githubusercontent.com/4998130/43232413-ddd622c2-902d-11e8-9cbe-965805698a5a.png)
Not sure where this "school" is coming from.

Reply:
school was the property name, now it is name, you can set the title to be 'Search school names' via `ui:title`, and remove the title from the custom field component, to achieve the desired appearance.